packages/components-js/projects/partials/scripts/generateInitialStylesPartial.ts

```diff
@@ -46,7 +53,7 @@ type GetInitialStylesOptionsWithoutTags = Omit<GetInitialStylesOptions, 'format'
         },
       },
 
-      '[data-theme="dark"] a:hover': {
+      'a[data-theme="dark"]': {
```

Comment:
Now it produces `a[data-theme="dark"]:hover{color:#ff0223}}` which isn't right, either?